devices/ubisys/h1_v1.7.json

```diff
@@ -0,0 +1,322 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "ubisys",
+  "modelid": "H1",
+  "vendor": "ubisys",
+  "product": "H1",
+  "sleeper": false,
+  "status": "Gold",
+  "path": "/devices/ubisys/h1_v1.7.json",
```

Comment:
Please remove the `path` stanza. Instead, please add:
`"matchexpr": "var v = R.item('attr/otaversion').val; (v == 0 || v >= 0x0170044D);"`

Reply:
This just does not work. I add this and my DDF is not selected. I remove it and it works again

Reply:
I removed it from the updated file to get it working for me again

Reply:
Oh, you got a firmware that is not in the archive and different than I recalled. That's why I asked for the screenshot. Entering that version from the screener should do the trick here.

Reply:
Yes, it looks like they released this version as 1.7.2. But shouldnt the expression match all versions greater than or equals to 1.7.0?

Reply:
> Yes, it looks like they released this version as 1.7.2. But shouldnt the expression match all versions greater than or equals to 1.7.0?

Ah yes, valid point. I just tested that mechanism for the very first time and it is indeed required to also touch the "old" DDF by adding the attribute as well and adding the below match expression
`"matchexpr": "var v = R.item('attr/otaversion').val; (v != 0 && v < 0x0170044D);"`

It is imperative not to mix that up 🙂